.github/workflows/no_byte_order_mark.yml

```diff
@@ -0,0 +1,16 @@
+name: Prevent ByteOrderMarks
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request: {}
```

Comment:
What is the purpose of these curvy brackets?

Reply:
to run/trigger on every pull request